packages/wallet-sdk/src/sdk-info.ts

```diff
@@ -0,0 +1,2 @@
+export const LIB_VERSION = '4.1.0';
```

Comment:
also, can we remove the `LIB_` prefix seems redundant

Reply:
removed LIB for both